src/Components/Common/PageTitle.tsx

```diff
@@ -70,15 +70,15 @@ export default function PageTitle({
             >
               <CareIcon
                 icon="l-angle-left"
-                className="border-box mr-1 rounded-md text-5xl hover:bg-secondary-200"
+                className="border-box mr-1 rounded-md text-5xl hover:bg-secondary-200 print:hidden"
```

Comment:
https://github.com/ohcnetwork/care_fe/pull/8441/files#r1739974547 is still not resolved.

Is this needed? We wouldn't be rendering the `PageTitle` component as child of `PrintPreview` component right? If we are, set the `print:hidden` class for the parent element instead of this reusable component.

```suggestion
                className="border-box mr-1 rounded-md text-5xl hover:bg-secondary-200"
```